pkg/integrations/node_exporter/config.go

```diff
@@ -0,0 +1,315 @@
+		"Regexp of fields to return for netstat collector.")
+	f.StringVar(&c.NTPServer, prefix+"ntp-server",
+		"127.0.0.1",
+		"NTP server to use for ntp collector")
+	f.IntVar(&c.NTPProtocolVersion, prefix+"ntp-protocol-version",
+		4,
+		"NTP protocol version")
+	f.BoolVar(&c.NTPServerIsLocal, prefix+"ntp-server-is-local",
+		false,
+		"Certify that collector.ntp.server address is not a public ntp server")
+	f.IntVar(&c.NTPIPTTL, prefix+"ntp-ip-ttl",
+		1,
+		"IP TTL to use while sending NTP query")
+	f.DurationVar(&c.NTPMaxDistance, prefix+"ntp-max-distance",
+		time.Microsecond*3466080,
+		"Max accumulated distance to the root")
+	f.DurationVar(&c.NTPLocalOffsetTolerance, prefix+"ntp-local-offset-tolerance",
+		time.Millisecond,
+		"Offset between local clock and local ntpd time to tolerate")
+	f.StringVar(&c.PerfCPUS, prefix+"perf-cpus",
+		"",
+		"List of CPUs from which perf metrics should be collected")
+	f.Var(&c.PerfTracepoint, prefix+"perf-tracepoint",
+		"perf tracepoint that should be collected for the perf collector")
+	f.StringVar(&c.PowersupplyIgnoredSupplies, prefix+"powersupply-ignored-supplies",
+		"^$",
+		"Regexp of power supplies to ignore for powersupplyclass collector.")
+	f.StringVar(&c.RunitServiceDir, prefix+"runit-service-dir",
+		"/etc/service",
+		"Path to runit service directory.")
+	f.StringVar(&c.SupervisordURL, prefix+"supervisord-url",
+		"http://localhost:9001/RPC2",
+		"XML RPC endpoint")
+	f.StringVar(&c.SystemdUnitWhitelist, prefix+"systemd-unit-whitelist",
+		".+",
+		"Regexp of systemd units to whitelist. Units must both match whitelist and not match blacklist to be included.")
+	f.StringVar(&c.SystemdUnitBlacklist, prefix+"systemd-unit-blacklist",
+		".+\\.(automount|device|mount|scope|slice)",
+		"Regexp of systemd units to blacklist. Units must both match whitelist and not match blacklist to be included.")
+	f.BoolVar(&c.SystemdEnableTaskMetrics, prefix+"systemd-enable-task-metrics",
+		false,
+		"Enables service unit tasks metrics unit_tasks_current and unit_tasks_max")
+	f.BoolVar(&c.SystemdEnableRestartsMetrics, prefix+"systemd-enable-restarts-metrics",
+		false,
+		"Enables service unit metric service_restart_total")
+	f.BoolVar(&c.SystemdEnableStartTimeMetrics, prefix+"systemd-enable-start-time-metrics",
+		false,
+		"Enables service unit metric unit_start_time_seconds")
+	f.StringVar(&c.VMStatFields, prefix+"vmstat-fields",
+		"^(oom_kill|pgpg|pswp|pg.*fault).*",
+		"Regexp of fields to return for vmstat collector.")
+	f.StringVar(&c.TextfileDirectory, prefix+"textfile-directory",
+		"",
+		"directory for the textfile collector to read from")
+
+}
+
+// MapConfigToNodeExporterFlags takes in a node_exporter Config and converts
+// it to the set of flags that node_exporter usually expects when running as a
+// separate binary.
+func MapConfigToNodeExporterFlags(c *Config) []string {
```

Comment:
Should we have a test for this function? A typo/change here end up in a breaking change.